src/shared/Core.UI.Avalonia/Controls/AboutWindow.axaml.cs

```diff
@@ -27,7 +40,7 @@ private void ProjectButton_Click(object sender, RoutedEventArgs e)
             {
                 UseShellExecute = true
             };
-            Process.Start(psi);
+            _processManager.CreateProcess(psi);
```

Comment:
Hmm... do we need to wrap this `Process.Start`? It's only launching a browser and we're not waiting for it end.

Plus just creating a `new Trace2()` instance means any child_start events are not going to go anywhere.

Reply:
I kind of just wrapped everything - went ahead and removed this, though!